manifests/0.5.0/3scale-community-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,306 @@
+    * **Upgrade** Upgrade from previously installed 3scale API Management solution
+    * **Reconcilliation** Tunable CRD parameters after 3scale API Management solution is installed
+    * **Capabilities** Ability to define 3scale API definitions and set them into a 3scale API Management solution
+
+    ### Documentation
+    Documentation can be found on our [website](https://github.com/3scale/3scale-operator/blob/v0.5.0/doc/operator-user-guide.md).
+
+    ### Getting help
+    If you encounter any issues while using 3scale operator, you can create an issue on our [website](https://github.com/3scale/3scale-operator) for bugs, enhancements, or other requests.
+
+    ### Contributing
+    You can contribute by:
+
+    * Raising any issues you find using 3scale Operator
+    * Fixing issues by opening [Pull Requests](https://github.com/3scale/3scale-operator/pulls)
+    * Improving [documentation](https://github.com/3scale/3scale-operator/blob/v0.5.0/doc/operator-user-guide.md)
+    * Talking about 3scale Operator
+
+    All bugs, tasks or enhancements are tracked as [GitHub issues](https://github.com/3scale/3scale-operator/issues).
+
+    ### License
+    3scale Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/3scale-operator/blob/master/LICENSE)
+  keywords:
+    - 3scale
+    - API
+  displayName: 3scale API Management
+  install:
+    spec:
+      deployments:
+      - name: 3scale-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: threescale-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: threescale-operator
+            spec:
+              containers:
+              - command:
+                - 3scale-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: threescale-operator
+                - name: BACKEND_IMAGE
+                  value: "quay.io/3scale/3scale28:apisonator-3scale-2.8.0-GA"
+                - name: APICAST_IMAGE
+                  value: "quay.io/3scale/3scale28:apicast-3scale-2.8.0-GA"
+                - name: SYSTEM_IMAGE
+                  value: "quay.io/3scale/3scale28:porta-3scale-2.8.0-GA"
+                - name: ZYNC_IMAGE
+                  value: "quay.io/3scale/3scale28:zync-3scale-2.8.0-GA"
+                - name: SYSTEM_MEMCACHED_IMAGE
+                  value: "memcached:1.5"
+                image: quay.io/3scale/3scale-operator:v0.5.0
+                name: 3scale-operator
+                resources: {}
+              serviceAccountName: 3scale-operator
+      permissions:
+      - rules:
```

Comment:
The rules section lacks the new addition of the poddisruptionbudget permissions.
Updated role permissions should be placed into the CSV

Reply:
This is because it is in `deploy/role.yaml`, but the PR updating `deploy/role.yaml` forgot to update `deploy/olm-catalog/3scale-operator-master/3scale-operator-master.v0.0.0.clusterserviceversion.yaml`

I will:
* fix in this PR 
* fix the CSV source file https://github.com/3scale/3scale-operator/blob/master/deploy/olm-catalog/3scale-operator-master/3scale-operator-master.v0.0.0.clusterserviceversion.yaml

Reply:
good catch

Reply:
updated in CSV source: https://github.com/3scale/3scale-operator/pull/326